src/calendar/actions/calendar-event/get-event.ts

```diff
@@ -0,0 +1,28 @@
+import { Fetcher } from '../../../common/api';
+import { endpoints } from '../../api/calendar-event/endpoints';
+import { ICalendarEvent } from '../../entities';
+import { ICalendarEventFilterDto } from '../../types';
+import { errorFactory, CalendarErr } from '../../errors/errors';
+import { StringUtils } from '../../../common/utils/string';
+
+export const getCalendarEventsById = async (
```

Comment:
```suggestion
export const getCalendarEventById = async (
```

Le pluriel n'est pas nécessaire ici comme la méthode retourne un seul élément.